packages/server/src/helpers/sanitize-address-middleware.js

```diff
@@ -0,0 +1,14 @@
+import { isAddress } from 'web3-utils'
+
+const sanitizeAddresses = object => {
+  Object.keys(object).forEach(key => {
```

Comment:
We could likely simplify this via `Object.entries()`, which will give us both the keys and values

Reply:
definitely!